packages/core/i18n/core-i18n-server-internal/src/init_translations.ts

```diff
@@ -11,10 +11,5 @@ import { i18n, i18nLoader } from '@kbn/i18n';
 export const initTranslations = async (locale: string, translationFiles: string[]) => {
   i18nLoader.registerTranslationFiles(translationFiles);
   const translations = await i18nLoader.getTranslationsByLocale(locale);
-  i18n.init(
-    Object.freeze({
-      locale,
-      ...translations,
-    })
-  );
+  i18n.init(Object.freeze({ ...translations }));
```

Comment:
So in https://github.com/elastic/kibana/pull/181650, I added translation file generation for the CDN, and I'm still passing the locale to `i18n.init` (because I need to call init again for each locale).

Is it going to be an issue?

Reply:
Ideally we'd wait on other i18n work until the system upgrade epic is complete, though it might take a bit to complete the full list so i'm not sure if the locales CDN issue can wait.

I went through the code and calling init multiple times should re-initiate i18n multiple times each time with the provided locale. So it should work.

I am just curious if we'd want to generate the cdn during the Kibana build for serverless, because we have a different release cycle for localization that doesnt have to be in sync with the kibana releases (for serverless we are thinking to do it on relatively regular intervals rather than on a per release basis.

Reply:
> I am just curious if we'd want to generate the i18n cdn assets during the Kibana build for serverless, because we have a different release cycle for localization that doesnt have to be in sync with the kibana releases

That's a fair question, and I came to the conclusion that it would still be better than not having those assets on the CDN at all:

The i18n "releases" are not directly synced with Kibana's release cycle, but each serverless release of Kibana has the latest set of translations it should have (we will never want a new serverless release to point to older translation file). 

So even if not perfect, I think this is still okay (and functionally not really worse than having a "dedicated" i18n release folder/release on the CDN)

wdyt?